lib/engine/action/assign.rb

```diff
@@ -8,7 +8,7 @@ class Assign < Base
       attr_reader :target
 
       def initialize(entity, target:)
-        @entity = entity
+        super(entity)
```

Comment:
why are all of these changes? i think it's probably too much change and we should rethink it

Reply:
That part was done by @dlpierce. I'm not sure why he did that

Reply:
This is so every action will have @created_at from Action::Base.